docs/elements.md

```diff
@@ -937,15 +937,16 @@ Provides an in-browser rich text editor, aimed mostly at manual grading essay-ty
+| `directory`          | string  | special                       | Directory where the source file with existing code is to be found. Only useful if `source-file-name` is used. If it contains one of the special names `clientFilesCourse` or `serverFilesCourse`, then the source file name is read from the course's special directories, otherwise the directory is expected to be in the question's own directory. If not provided, the source file name is expected to be found in the question's main directory. |
+| `placeholder`        | string  | "Your answer here"            | Text to be shown in the editor as a placeholder when there is no student input.                                                                                                                                                                                                                                                                                                                                                                       |
+| `format`             | string  | `html` or `markdown`          | Format used to save the student's response. This format also affects how the source file name or inner HTML is interpreted.                                                                                                                                                                                                                                                                                                                           |
+| `markdown-shortcuts` | boolean | `true`                        | Whether or not the editor accepts shortcuts based on markdown format (e.g., typing `_word_` causes the word to become italic).                                                                                                                                                                                                                                                                                                                        |
+| `counter`            | string  | `word`, `character` or `none` | Display a `word` or `character` count.                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
Note that the second column is the type, and the third is the default value.
```suggestion
| `format`             | `html` or `markdown` | `html` | Format used to save the student's response. This format also affects how the source file name or inner HTML is interpreted.                                                                                                                                                                                                                                                                                                                           |
| `markdown-shortcuts` | boolean | `true`                        | Whether or not the editor accepts shortcuts based on markdown format (e.g., typing `_word_` causes the word to become italic).                                                                                                                                                                                                                                                                                                                        |
| `counter`            | `word`, `character` or `none` | `none` | Whether a word or character count should be displayed at the bottom of the editor.                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Again, this will require `make format`.